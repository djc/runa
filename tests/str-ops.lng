def __main__(name: str, args: list[str]) -> int:
	print('a' + 'b')
