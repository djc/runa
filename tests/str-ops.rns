def main():
	print('' if 'a' == 'b' else 'X')
	print('' if 'aa' == 'a' else 'X')
	print('X' if 'aaa' == 'aaa' else '')
	print('X' if 'a' < 'b' else '')
	print('' if 'b' < 'a' else 'X')
	print('a' + 'b')
	print('test'.__copy__())
