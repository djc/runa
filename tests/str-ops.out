X
X
X
X
X
ab
